src/SFA.DAS.AssessorService.Data/Apply/ApplyRepository.cs

```diff
@@ -91,34 +91,42 @@ GROUP BY
                 transaction: _unitOfWork.Transaction)).ToList();
         }
 
-        public async Task<List<ApplySummary>> GetAllWithdrawnApplicationsForStandard(Guid orgId, int? standardCode)
-        {
-            var query = $@"SELECT * from [dbo].[Apply] 
-                        WHERE organisationId = @orgId 
-                        AND StandardCode = @standardCode 
-                        AND ReviewStatus = '{ApplicationStatus.Approved}' 
-                        AND (standardApplicationType = '{StandardApplicationTypes.StandardWithdrawal}' 
-                        OR standardApplicationType = '{StandardApplicationTypes.VersionWithdrawal}') 
-                        ORDER BY CreatedAt DESC";
-
-            return (await _unitOfWork.Connection.QueryAsync<ApplySummary>(
+        /// <summary>
+        /// Gets the lastest withdrawal date for the given standard by the given organisation.
+        /// </summary>
+        /// <remarks>
+        /// This does not include where the standard is withdrawn because the organisation has withdrawn
+        /// and the standard was assessed by the organisation at the time the organisation withdrew as
+        /// the database does not hold a complete enough audit trail to be able to determine that reliably.
+        /// 
+        /// This method is subject to Tech Debt QF-1607 in which the database may store the date of the
+        /// approval of withdrawal for a standard or the EffectiveTo date may be used instead
+        /// </remarks>
+        /// <param name="organisationId">The organisation from which the standard has been withdrawn</param>
+        /// <param name="standardCode">The standard which was withdrawn</param>
+        /// <returns></returns>
+        public async Task<DateTime?> GetLatestWithdrawalDateForStandard(Guid organisationId, int? standardCode)
```

Comment:
I think the QF-1607 ticket should contain some information about this. I've added a stub for now

Reply:
Ok thanks